lib/build.js

```diff
@@ -294,13 +294,14 @@ export default {
           success = success && !this.errorMatcher.hasMatch();
         }
 
+        const path = require('path');
         this.linter && this.linter.setMessages(this.errorMatcher.getMatches().map(match => ({
           type: 'Error',
           text: match.message || 'Error from build',
-          filePath: match.file,
+          filePath: path.isAbsolute(match.file) ? match.file : path.join(atom.project.getPaths()[0], match.file),
```

Comment:
The error match file/configuration is not necessarily in the first project. The path you're actually looking for here is in the variable `p` (determined at the top of this function).